lsp/texlab.lua

```diff
@@ -193,29 +169,17 @@ return {
     },
   },
   on_attach = function(_, buf)
-    vim.api.nvim_buf_create_user_command(buf, 'LspTexlabBuild', client_with_fn(buf_build), {
-      desc = 'Build the current buffer',
-    })
-    vim.api.nvim_buf_create_user_command(buf, 'LspTexlabForward', client_with_fn(buf_search), {
-      desc = 'Forward search from current position',
-    })
-    vim.api.nvim_buf_create_user_command(buf, 'LspTexlabCancelBuild', client_with_fn(buf_cancel_build), {
-      desc = 'Cancel the current build',
-    })
-    vim.api.nvim_buf_create_user_command(buf, 'LspTexlabDependencyGraph', client_with_fn(dependency_graph), {
-      desc = 'Show the dependency graph',
-    })
-    vim.api.nvim_buf_create_user_command(buf, 'LspTexlabCleanArtifacts', client_with_fn(command_factory('Artifacts')), {
-      desc = 'Clean the artifacts',
-    })
-    vim.api.nvim_buf_create_user_command(buf, 'LspTexlabCleanAuxiliary', client_with_fn(command_factory('Auxiliary')), {
-      desc = 'Clean the auxiliary files',
-    })
-    vim.api.nvim_buf_create_user_command(buf, 'LspTexlabFindEnvironments', client_with_fn(buf_find_envs), {
-      desc = 'Find the environments at current position',
-    })
-    vim.api.nvim_buf_create_user_command(buf, 'LspTexlabChangeEnvironment', client_with_fn(buf_change_env), {
-      desc = 'Change the environment at current position',
-    })
+    for _, command in ipairs({
+      { name = 'LspTexlabBuild', fn = buf_build, desc = 'Build the current buffer' },
+      { name = 'LspTexlabForward', fn = buf_search, desc = 'Forward search from current position' },
+      { name = 'LspTexlabCancelBuild', fn = buf_cancel_build, desc = 'Cancel the current build' },
+      { name = 'LspTexlabDependencyGraph', fn = dependency_graph, desc = 'Show the dependency graph' },
+      { name = 'LspTexlabCleanArtifacts', fn = command_factory('Artifacts'), desc = 'Clean the artifacts' },
+      { name = 'LspTexlabCleanAuxiliary', fn = command_factory('Auxiliary'), desc = 'Clean the auxiliary files' },
+      { name = 'LspTexlabFindEnvironments', fn = buf_find_envs, desc = 'Find the environments at current position' },
+      { name = 'LspTexlabChangeEnvironment', fn = buf_change_env, desc = 'Change the environment at current position' },
+    }) do
+      vim.api.nvim_buf_create_user_command(buf, command.name, client_with_fn(command.fn), { desc = command.desc })
```

Comment:
To avoid the (naive :) lint-rule could you do something like:


```suggestion
      vim.api.nvim_buf_create_user_command(buf, 'Lsp' .. command.name, client_with_fn(command.fn), { desc = command.desc })
```